### ISSUE TITLE
[TSDoc> Search]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for the 'Search' text.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/rushstack-websites#141]
**Test Environment:**
OS Version: 22H2 (OS Build 25370.1)
Browser: Edge Dev (Version 115.0.1866.1 (Official build) dev (64-bit))
URL: https://tsdoc.org/
Tool: Accessibility Insights for Web

**Repro Steps:**
1. Open the above URL in edge dev browser and login with v-.
2. Run the fast pass of accessibility insights for web.
3. Observe the issue on 'Search' edit field.

**Actual Result:** 
Contrast between foreground and background colors does not meets WCAG 2 AA contrast ratio thresholds for the 'Search' text.
Luminosity ratio for the 'Search' placeholder text under 'Search' edit field is 2.666:1 which is less than 4.5:1.

**Expected Result:** 
The contrast between foreground and background colors must meets WCAG 2 AA contrast ratio thresholds for the 'Search' text.
Luminosity ratio for the 'Search' placeholder text under 'Search' edit field should be greater than or equals to 4.5:1.

**Snippet:** 
<span class="DocSearch-Button-Placeholder">Search</span>

**How to fix:** 
Fix any of the following:
  Element has insufficient color contrast of 2.66 (foreground color: #969faf, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

**Notes:**
1. Same issue repro for code lines present under 'export class Statistic , get Average, IsIineTag and the code lines present under 'Consider a hypothetical input' text.
2. Same issue repro on code line present under 'Example' heading of 'Tags' link under main navigation landmark.
3. Same issue repro on code line present under 'Example' heading of '@returns' link under 'docs sidebar navigation landmark'.
4. Same issue repro on #Run this ....$rush install, $npm text in 3 bullet point, $rush change in the 5 bullet point in 'Submitting a pull request link under 'docs sidebar navigation landmark'.
5. Same issue repro on 2023 Microsoft text under footer landmark and this issue is throughout the page.

**User Impact:**
Person with visual disability will get impacted as user will not be able to see the text clearly and hence will not be able to interact with the content easily.

**WCAG Reference Link:**
https://www.w3.org/TR/WCAG21/#contrast-minimum

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

<img width="960" alt="MAS1 4 3 Luminosity ratio of Search placeholder is less than required" src="https://github.com/microsoft/rushstack-websites/assets/104483005/51e97d16-68a5-4e1a-a610-dc674dc8f595">
<img width="951" alt="MAS1 4 3 Luminosity ratio of search placeholder text is less than required" src="https://github.com/microsoft/rushstack-websites/assets/104483005/9418b399-ea01-4353-b9de-033a39dfddee">
<img width="960" alt="Notes~ Contrast ratio is less than required" src="https://github.com/microsoft/rushstack-websites/assets/104483005/2e411ebe-6ce4-4f70-992a-e989fffb99e7">

**Comment by vaishali1397** [comment]
Closing the issue and will track with bug id:
https://github.com/microsoft/rushstack-websites/issues/166

**Comment by vaishali1397** [comment]
Closing this issue and will track with bug id: https://github.com/microsoft/rushstack-websites/issues/166
